config/initializers/sentry.rb

```diff
@@ -4,6 +4,10 @@
     config.breadcrumbs_logger = [:active_support_logger, :http_logger]
     config.environment = HostEnv.env_name
 
+    # Opt in to new Rails error reporting API
+    # https://edgeguides.rubyonrails.org/error_reporting.html
+    config.rails.register_error_subscriber = true
```

Comment:
I thought this was the default. "Some error-reporting libraries, such as [Sentry's](https://github.com/getsentry/sentry-ruby/blob/e18ce4b6dcce2ebd37778c1e96164684a1e9ebfc/sentry-rails/lib/sentry/rails/error_subscriber.rb) and [Honeybadger's](https://docs.honeybadger.io/lib/ruby/integration-guides/rails-exception-tracking/), automatically register a subscriber for you. Consult your provider's documentation for more details."

Reply:
Yep I thought so too, but they retracted their "default" position, and made it opt-in.
PR for reference:
https://github.com/getsentry/sentry-ruby/pull/1771

They seem to be still working out some kinks before making it default again.